api/RTCPeerConnectionIceEvent.json

```diff
@@ -0,0 +1,203 @@
+            {
+              "alternative_name": "RTCIceCandidateEvent",
+              "version_added": true
+            },
+            {
+              "version_added": "56"
+            }
+          ],
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": true
+          },
+          "firefox_android": {
+            "version_added": null
+          },
+          "ie": {
+            "version_added": null
+          },
+          "opera": {
+            "version_added": true
+          },
+          "opera_android": {
+            "version_added": true
+          },
+          "safari": {
+            "version_added": null
+          },
+          "safari_ios": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "candidate": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCPeerConnectionIceEvent/candidate",
+          "support": {
+            "webview_android": [
+              {
+                "alternative_name": "RTCIceCandidateEvent",
+                "version_added": true
+              },
+              {
+                "version_added": "56"
+              }
+            ],
+            "chrome": [
+              {
+                "alternative_name": "RTCIceCandidateEvent",
+                "version_added": true
+              },
+              {
+                "version_added": "56"
+              }
+            ],
+            "chrome_android": [
+              {
+                "alternative_name": "RTCIceCandidateEvent",
+                "version_added": true
+              },
+              {
+                "version_added": "56"
+              }
+            ],
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": true
+            },
+            "firefox_android": {
+              "version_added": null
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": true
+            },
+            "opera_android": {
+              "version_added": true
+            },
+            "safari": {
+              "version_added": null
+            },
+            "safari_ios": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "RTCPeerConnectionIceEvent": {
+        "__compat": {
+          "description": "<code>RTCPeerConnectionIceEvent()</code>",
```

Comment:
Add constructor to make it clearer:
    "description": "\<code>RTCPeerConnectionIceEvent()\</code> constructor",